src/main/kotlin/exception/NameException.kt

```diff
@@ -0,0 +1,28 @@
+package exception
+
+import message.ExceptionMessage
+
+class NameException {
+    private val nullException = NullException()
+    fun checkNames(carNames: String) {
+        splitCarNames(carNames).forEach { name ->
+            checkNameRange(nullException.checkNull(name))
```

Comment:
```suggestion
            checkNameRange(name)
```
checkNames 함수의 파라미터(name)의 타입은 non-null 타입입니다.
즉, name은 null일 수 없다는 의미입니다.